src/Tizen.NUI/src/public/Rendering/RendererConstants.cs

```diff
@@ -0,0 +1,465 @@
+﻿// Copyright (c) 2019 Samsung Electronics Co., Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// Enumeration for face culling mode.
+    /// </summary>
+    /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum FaceCullingModeType
```

Comment:
Just my curiosity :
Is it okay to open these all enumerations as public? Because they are originally internal values in NUI.
If they're public APIs in DALi, I think it's available. :)

Reply:
All of enum values (instead of some `BlendEquationType` cases) are public. We can make them as public.
Actually, they don't have any reason to be internal :(....

Reply:
I see. All the enum values are in internal folders, but they can be public and users are able to use the enum directly.